src/adapter/graphql/src/queries/flows/flow_start_condition.rs

```diff
@@ -43,13 +43,19 @@ impl From<fs::FlowStartConditionThrottling> for FlowStartConditionThrottling {
 
 #[derive(SimpleObject)]
 pub(crate) struct FlowStartConditionBatching {
-    pub threshold_new_records: usize,
+    pub active_batching_rule: FlowConfigurationBatching,
+    pub awaited_by_now: TimeDelta,
```

Comment:
Why not track `batch_started_at` time instead?

Or if we require `max_interval` then this field could become something like `batch_deadline`.

Either way I think having absolute time in API is much nicer than relative deltas that are different on each request.

Reply:
Ended with `batching_deadline`.